pkg/asset/agent/image/ignition_test.go

```diff
@@ -296,6 +296,37 @@ func TestAddHostConfig_Roles(t *testing.T) {
 
 }
 
+func defaultGeneratedFiles() []string {
```

Comment:
Will we have to update this list every time we add a file to the image?

I think we should only test for files in the extra-manifests directory so that we can avoid that.

Reply:
TBH I did it intentionally, as I noticed that we didn't have any coverage on the files we're going to add in the ISO (the "core" business for this assets) to catch any regression, which is critical point.
In this way, if in future someone will remove one portion of the code related to one or more files, then at least it would be signaled by a failing test (even better to check the content, but that would require more effort)